Mathlib/Logic/Equiv/Basic.lean

```diff
@@ -374,6 +374,21 @@ theorem sumCongr_refl : sumCongr (Equiv.refl α) (Equiv.refl β) = Equiv.refl (S
 
 end Perm
 
+section LiftRel
+
+variable {r : α → γ → Prop} {s : β → δ → Prop} {a : α} {b : β} {c : γ} {d : δ}
+  {x : Sum α β} {y : Sum γ δ} {e : Sum α β ≃ Sum γ δ} {f : Sum γ δ ≃ Sum α β}
```

Comment:
Putting e and f in the variable line seems a bit unnecessary if they are used only once. Its probably clearer to someone reading the file not to do this